app/assets/styles/core/_base.scss

```diff
@@ -33,11 +33,17 @@ html {
 }
 
 body {
-  background: tint($base-color, 98%);
-  color: $base-font-color;
+  @if $production_environment == true {
+    background: tint($base-color, 98%);
+    color: $base-font-color;
+    font-family: $base-font-family;
+  } @else {
+    background: #88AADD;
+    color: #0000FF;
+    font-family: "Cambria, Georgia, serif";
```

Comment:
@szabozoltan69 I don't think you should change the font family under dev, a different color would be enough, no?

Also, I think it's better to control this from the javascript, where you have more knowledge of what environment something is in.

Reply:
Ok, any other solution is welcome. I thought to change font family because it gives a basic style, "feeling", and the aim was to see at once that something is different here. But it was just an idea, we can work with colors only, if nicer.